docs/source/plugin-development/registering-plugins.rst

```diff
@@ -112,6 +112,9 @@ look like::
 
     X101 = flake8_example:ExamplePlugin
 
+In the above, ``X`` is the entry point prefix and ``X101`` is the
+entry point name.
```

Comment:
ah so actually both `X` and `X101` are used as the entry point name (which *is* the prefix) -- the error code is `X101`

Reply:
Looks like I wasn't fully understanding that still.  I was looking at "ABC" and "123" as 2 different parts of the entry point name.  I saw how the combination was the error code (or error code "prefix" in the case of something like X1 or just X).  I added this to try to capture your comment about 3 letters being the max prefix length - I was trying to define what the "prefix" meant for an entry point.  Would "In the above, you can think of the letters as a prefix to the entry point name." be better or maybe just leave those two lines out?

Reply:
the entry point is the prefix of the codes that the plugin produces -- I think that's the important part from this section

Reply:
Ah - that's a much cleaner way to express it!  I'll push an update that (hopefully) reflects that cleanly.